modules/central-configuration-options-operator.adoc

```diff
@@ -75,6 +75,40 @@ If no PVC with the given name exists, it will be created. The default value is `
 |`central.imagePullSecrets`
 |Use this parameter to specify the image pull secrets for the Central image.
 
+| `central.db.passwordSecret.name`
+| Specify a secret that contains the database password in the `password` data item. This can only be used when specifying a connection string manually. When omitted, the operator autogenerates a password and stores it in the `password` item in the `central-db-password` secret.
+
+|`central.db.connectionString`
+a| *Tech Preview*: Setting this parameter will not deploy Central DB and Central will connect via the specified connection string. `central.db.passwordSecret.name` must be specified if this field is populated.
+Constraints:
+
+* Connection string must be in keyword/value format as described in https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING[PostgreSQL Docs]
+* Only Postgres 13 is supported.
+* Connections through PGBouncer are not supported.
+* User must be superuser with ability to create and delete databases.
+
+| `central.db.tolerations`
+| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for Central DB. This parameter is mainly used for infrastructure nodes.
+
+|`central.db.persistence.hostPath.path`
+|Specify a host path to store persistent data in a directory on the host. Red Hat does not recommend using this. If you need to use host path, you must use it with a node selector.
+
+|`central.db.persistence.persistentVolumeClaim.claimName`
+|The name of the PVC to manage persistent data.
+If no PVC with the given name exists, it will be created. The default value is `central-db` if not set. To prevent data loss the PVC is not removed automatically with Central DB`s deletion.
```

Comment:
```suggestion
If no PVC with the given name exists, it will be created. The default value is `central-db` if not set. To prevent data loss the PVC is not removed automatically with Central DB's deletion.
```